src/inmanta/deploy/tasks.py

```diff
@@ -182,6 +183,11 @@ async def execute(self, task_manager: "scheduler.TaskManager", agent: str, reaso
             # Deploy
             try:
                 deploy_result = await my_executor.execute(action_id, gid, executor_resource_details, reason, requires)
+                # Translate deploy result status to the new deployment result state
+                if deploy_result.status == const.ResourceState.deployed:
+                    deploy_state = state.DeploymentResult.DEPLOYED
+                elif deploy_result.status == const.ResourceState.skipped:
+                    deploy_state = state.DeploymentResult.SKIPPED
```

Comment:
Purely a style preference, but I think this would be more readable as a conditional expression.

Reply:
(100% optional)